mono/metadata/object.h

```diff
@@ -5,6 +5,7 @@
 #ifndef _MONO_CLI_OBJECT_H_
 #define _MONO_CLI_OBJECT_H_
 
+#include <mono/metadata/object-forward.h>
```

Comment:
Same problem as the other two headers.

Reply:
Right, should be fixed by 3b892d45b000f7e9d4ec988dd00beebede1afe9d.